data/json/recipes/recipe_deconstruction.json

```diff
@@ -938,580 +938,547 @@
       [ [ "RAM", 1 ] ],
       [ [ "scrap", 1 ] ]
     ]
-},{
-  "type" : "recipe",
-  "result": "broken_grenade_hack",
-  "category": "CC_NONCRAFT",
-  "subcategory": "CSC_NONCRAFT",
-  "skill_used": "electronics",
-  "difficulty": 4,
-  "time": 2000,
-  "reversible": true,
-  "autolearn": false,
-  "qualities" : [
-    {"id":"SCREW","level":1,"amount":1}
-  ], "tools": [
-    [
-      [ "soldering_iron", 5 ],
-      [ "toolset", 5 ]
-    ]
-  ],
-  "components": [
-    [
-      [ "processor", 1 ]
-    ],
-    [
-      [ "RAM", 1 ]
+  }, 
  {
+    "result": "broken_grenade_hack",
+    "type": "recipe",
+    "category": "CC_NONCRAFT",
+    "subcategory": "CSC_NONCRAFT",
+    "skill_used": "electronics",
+    "difficulty": 4,
+    "time": 2000,
+    "reversible": true,
+    "qualities" : [ {"id": "SCREW","level": 1 } ],
+    "tools": [
+      [ [ "soldering_iron", 5 ] ],
+      [ ["toolset", 5 ] ]
     ],
```

Comment:
Same error as with copper tubing: you're changing it to require both the soldering iron and toolset, not one or other.